contrib/inventory_builder/inventory.py

```diff
@@ -81,16 +97,17 @@ def __init__(self, changed_hosts=None, config_file=None):
                 self.yaml_config = yaml.load(self.hosts_file)
             except FileNotFoundError:
```

Comment:
Can you change this to except `IOError:`
The tests now run in python3 and there is no such built in exception and it would never get called.

Reply:
done.